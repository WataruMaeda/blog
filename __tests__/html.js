import React from 'react'
import renderer from 'react-test-renderer'
import HTML from '../src/html'

describe('HTML', () => {
  it('renders correctly', () => {
    const props = {
      htmlAttributes: {},
      headComponents: [],
      bodyAttributes: {},
      preBodyComponents: [],
      body: '',
      postBodyComponents: [],
    }
    const tree = renderer.create(<HTML {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
