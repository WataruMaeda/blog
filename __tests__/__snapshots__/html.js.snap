// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTML renders correctly 1`] = `
<html
  lang="ja"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <meta
      content="ie=edge"
      httpEquiv="x-ua-compatible"
    />
    <meta
      content="width=device-width, initial-scale=1, shrink-to-fit=no"
      name="viewport"
    />
    <link
      href="opensearch.xml"
      rel="search"
      type="application/opensearchdescription+xml"
    />
  </head>
  <body>
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "",
        }
      }
      id="___gatsby"
    />
  </body>
</html>
`;
